src/analysis/seminorm.lean

```diff
@@ -708,6 +708,71 @@ section normed_field
 variables [normed_field 𝕜] [add_comm_group E] [module 𝕜 E] (p : seminorm 𝕜 E) {A B : set E}
   {a : 𝕜} {r : ℝ} {x : E}
 
+lemma l (x : E) (p q : seminorm 𝕜 E) : bdd_below (range (λ (u : E), p u + q (x - u))) :=
+begin
+  use 0, intro, intro h, cases h, rw ← h_h,
+  dsimp, apply add_nonneg, apply nonneg, apply nonneg
+end
```

Comment:
```suggestion
by { use 0, rintro _ ⟨x, rfl⟩, exact add_nonneg (p.nonneg _) (q.nonneg _) }
```

Reply:
This lemma doesn't deserve to exist IMO

Reply:
I wrote a lemma to avoid rewriting every time its proof, as I need it every time I want to say something about the infi

Reply:
You can make it `private`.